### ISSUE TITLE
Letter to Scientific Community

### ISSUE THREAD

**Issue by REFATESHAQ** [REFATESHAQ/PIDC-NN_MinerNet#5]
Regardless of the high accuracy of deep learning-based segmentation (DLS) and deep neural networks DNNs that astonish the scientific community as they claim, the adoption of these techniques in such medical field or industry field is still questionable and challenging because DLS and DNNs have been built based on simple mathematical calculation. Moreover, from methodology, the DLS andDNNs are a simple combination of convolutional layers, nonlinear activation functions layers, batch normalization layers, polling layers, and dropout layers, which put forward many questions about their effectiveness in other different fields. Also, it has been reported that most computer scholars and researchers tend to steal the ideas of mathematicians and biologists and then design algorithms based on the basic knowledge of mathematics and biology, after that, they attribute the proposed algorithms in their favor in order to obtain fame and money which consider the immoral act and lead to scientific disaster. It's surprising that their works have been published in top journals such as IEEE Transactions, ACM Transactions, IEEE conferences, and other computer journals in IEEE and ACM
